app/src/main/java/net/squanchy/speaker/SpeakerDetailsActivity.kt

```diff
@@ -42,9 +43,11 @@ class SpeakerDetailsActivity : AppCompatActivity() {
 
     private fun setupToolbar(toolbar: Toolbar) {
         setSupportActionBar(toolbar)
-        supportActionBar!!.setDisplayShowTitleEnabled(false)
-        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
-        supportActionBar!!.setHomeAsUpIndicator(R.drawable.ic_close)
+        supportActionBar?.apply {
```

Comment:
uhm why `?`

Reply:
Because it's `@Nullable` — I'm replacing the `!!`s with one `?.apply`

Reply:
Tell that to Android